clang-tools-extra/clangd/ClangdServer.h

```diff
@@ -260,6 +263,8 @@ class ClangdServer {
   void locateSymbolAt(PathRef File, Position Pos,
                       Callback<std::vector<LocatedSymbol>> CB);
 
+  void findAST(const SearchASTArgs &Args, Callback<BoundASTNodes> CB);
```

Comment:
The convention at this layer seems to be to pass the input parameters separately, so something like:

```
void findAST(PathRef File, llvm::StringRef SearchQuery, Callback<BoundASTNodes> CB);
```